addons/editor-compact/userstyle.css

```diff
@@ -0,0 +1,492 @@
+}
+[class*="sprite-info_row"] {
+  justify-content: flex-start;
+}
+[class*="sprite-info_group"] {
+  margin-right: 1rem;
+}
+[class*="sprite-info_icon-wrapper"] {
+  width: 1.5rem;
+  height: 1.5rem;
+  padding: 0;
+}
+[class*="sprite-info_icon-wrapper"] img {
+  padding: 0.25rem;
+}
+[class*="sprite-info_sprite-info"] [class*="input_input-form"],
+[class*="gui_tabs"] [class*="input_input-form"] {
+  height: 1.5rem;
+  padding: 0 0.5rem;
+}
+[class*="sprite-selector-item_is-selected"] [class*="sprite-selector-item_sprite-info"] {
+  background: none;
+  color: inherit;
+}
+[class*="label_input-group"] [class*="input_input-small"],
+[class*="label_input-group"] [class*="sprite-info_larger-input"] {
+  width: 2.5rem !important;
+}
+
+[class*="sprite-selector_scroll-wrapper"] {
+  height: calc(100% - 4.75rem);
+}
+[class*="sprite-selector_items-wrapper"] {
+  padding: 0.125rem;
+}
+[class*="sprite-selector_sprite-wrapper"] {
+  margin: 0.125rem;
+}
+[class*="sprite-selector-item_delete-button"] {
+  top: -0.25rem;
+}
+[dir="ltr"] [class*="delete-button_delete-button_"] {
+  right: -0.25rem;
+}
+[dir="rtl"] [class*="delete-button_delete-button_"] {
+  left: -0.25rem;
+}
+[class*="delete-button_delete-button-visible"] {
+  width: 1rem;
+  height: 1rem;
+}
+[class*="delete-button_delete-icon"] {
+  width: 2rem;
+  height: 2rem;
+  bottom: 0.25rem;
+}
+[class*="sprite-selector-item_sprite-image-inner"] {
+  margin: 0;
+}
+img[class*="sprite-selector-item_sprite-image"] {
+  max-width: 56px;
+  max-height: 32px;
+}
+
+[class*="action-menu_main-button"] {
+  width: 2rem;
+  height: 2rem;
+}
+[class*="action-menu_main-button"] img {
+  width: 1.5rem;
+  height: 1.5rem;
+}
+[class*="action-menu_more-buttons-outer"] {
+  width: 2rem;
+  bottom: 2rem;
+  padding-bottom: calc(2rem / 2);
+  margin: 0 0 calc(2rem / -2) 0;
+  border-top-left-radius: 2rem;
+  border-top-right-radius: 2rem;
+}
+button[class*="action-menu_more-button"] {
+  width: 2rem;
+  height: 2rem;
+}
+
+/* Tabs area */
+[class*="gui_tab-list"] {
+  height: 2rem;
+}
+[class*="gui_tab_"] {
+  padding-left: 1rem;
+  padding-right: 1rem;
+  border-radius: 0.5rem 0.5rem 0 0;
+}
+[class*="gui_tab_"] > img {
+  display: none;
+}
+
+#s3devToolBar {
+  align-self: center;
+}
+.s3devLabel {
+  align-items: center;
+}
+#s3devFind {
+  height: 20px;
+}
+#s3devHelpContent {
+  height: calc(100% - 2rem);
+}
+
+/* Code tab */
+[iscolumns="true"] .blocklyToolboxDiv,
+.blocklyFlyout {
+  width: 210px;
+}
+[class*="gui_tab-panel"]:not([iscolumns="true"]) .sa-flyout-placeHolder {
+  width: calc(210px + 1px);
+}
+.sa-lock-image {
+  left: calc(210px - 30px);
+}
+.blocklyScrollbarVertical.blocklyFlyoutScrollbar {
+  transform: translate(calc(196.5px + 60px), 2.5px) !important;
+}
+[iscolumns="true"] .blocklyScrollbarVertical.blocklyFlyoutScrollbar {
+  transform: translate(196.5px, 172.5px) !important;
+}
+.scratchCategoryItemBubble,
+.scratchCategoryItemIcon {
+  width: 0.75rem;
+  height: 0.75rem;
```

Comment:
This makes icons really tiny. Reducing padding might be a better way to make the category menu compact.